src/app/page.module.css

```diff
@@ -1,9 +1,61 @@
-.main {
+.mainpageContainer {
+  background: #1b1228;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+
+.welcomecontainer {
+  width: 353px;
+  height: 92px;
+  margin-top: 1px;
+  maring-left: 19px;
+  gap: 0px;
+  opacity: 0px;
+  font-family: Cascadia Mono;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 46.48px;
+  text-align: center;
+  color: #ffffff;
```

Comment:
delete, font color is defined else where and cant be defined here for the sake of switching between light/dark mode